src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -125,6 +128,38 @@ public static Project parseProject(String project) throws ParseException {
         return new Project(trimmedProject);
     }
 
+    /**
+     * Parses a {@code String taskName} into a {@code Task}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code Task} is invalid.
+     */
+    public static Task parseTask(String taskName, String deadlineString) throws ParseException {
+        requireNonNull(taskName, deadlineString);
+        String trimmedTaskName = taskName.trim();
+        if (!Project.isValidProject(trimmedTaskName)) {
```

Comment:
I believe it should be `Task.isValidTask(trimmedTaskName)`?

Reply:
yup